v3/tutorials/09-cumulus/c-polkadot-launch/index.mdx

```diff
@@ -0,0 +1,344 @@
+---
+title: Launch a Parachain Testnet
+slug: /tutorials/v3/cumulus/pt3
+version: 'polkadot-0.9.10'
+section: tutorials
+category: parachains
+duration: 3 Hours
+difficulty: 3
+keywords: cumulus, relay chain, parachain, parathread, paraid, template, rococo, xcm, xcmp, collator
```

Comment:
```suggestion
keywords:
  [
    cumulus,
    tutorial,
    workshop,
    parachain,
    parathread,
    paraid,
    register,
    template,
    rococo,
    parathread,
    xcm,
    xcmp,
    relay chain,
    collator,
  ]
```
 cc @jimmychu0807 you reverted this change.
This works on my end, and IMHO the more keywords the better for the search engine to pick this up in context, right?